pkg/systemd/logs.scss

```diff
@@ -47,7 +47,6 @@
     .multiline {
         word-break: break-all;
         white-space: pre-wrap !important;
-        font-family: monospace, monospace;
```

Comment:
This applies to both pf-c-card__title (where it's justified and matches the intention) and .multiline, which we use in various ABRT logs. It should probaly stay for .multiline, i.e. this rule needs to be broken up?

Reply:
I brought it back for the multine class.